test.js

```diff
@@ -82,3 +82,15 @@ test('bits option', t => {
 	t.is(prettyBytes(1e16, {bits: true}), '10 Pbit');
 	t.is(prettyBytes(1e30, {bits: true}), '1000000 Ybit');
 });
+
+test('binary option', t => {
+	t.is(prettyBytes(0, {binary: true}), '0 B');
+	t.is(prettyBytes(4, {binary: true}), '4 B');
+	t.is(prettyBytes(10, {binary: true}), '10 B');
+	t.is(prettyBytes(10.1, {binary: true}), '10.1 B');
+	t.is(prettyBytes(999, {binary: true}), '999 B');
+	t.is(prettyBytes(1025, {binary: true}), '1 kB');
```

Comment:
shouldn't this result in `1 kiB`?

Reply:
Only if both `binary: true` and `bits: true`.

Reply:
I don't understand. Why are `bits` involved? shouldn't it be `kiBit` then?

Reply:
and from the README:

```js
prettyBytes(1024, {binary: true});
//=> '1 kiB'
```

there is no mention of `bits: true` there 🤔

Reply:
I see what you mean. I have fixed it with #61. Thanks!